### PR TITLE
All CI tests pass, dapps are updated

### LINKED ISSUE #3692: Metering: clean up bootstrap and dapps
The last finishing touches on #3294 are ensuring that the bootstrap process is set up correct and with the correct settings, and that the dapps are all updated to the new `makeZoeKit` signature.

### PR DESCRIPTION
**Pull request by katelynsills**
Closes #3692 

Stacked on https://github.com/Agoric/agoric-sdk/pull/3690

This PR cleans up any remaining issues and ensures that all of the CI tests pass.

#getting-started-branch: 3294-zoe-metering
#dapp-card-store-branch: 3294-zoe-metering
#dapp-fungible-faucet-branch: 3294-zoe-metering
#dapp-otc-branch: 3294-zoe-metering
#dapp-oracle-branch: 3294-zoe-metering
#documentation-branch: 3294-zoe-metering

### CODE REVIEW

File: packages/dapp-svelte-wallet/api/src/lib-wallet.js [hunk 1]
```diff
@@ -591,7 +591,9 @@ export function makeWallet({
     const { brand } = await recP;
     if (!issuerToBoardId.has(issuer)) {
       const issuerBoardId = await E(board).getId(issuer);
-      issuerToBoardId.init(issuer, issuerBoardId);
+      if (!issuerToBoardId.has(issuer)) {
```
Comment (katelynsills) [inline packages/dapp-svelte-wallet/api/src/lib-wallet.js]: Hmm, is this check needed twice?

File: packages/dapp-svelte-wallet/api/src/lib-wallet.js [hunk 2]
```diff
@@ -1636,6 +1643,13 @@ export function makeWallet({
         }
       },
     }).finally(() => console.error('/// This is the end of the bank assets'));
+    if (!feePurseP) {
+      return;
+    }
+    const feePurse = await feePurseP;
+    const feeIssuer = await E(zoe).getFeeIssuer();
+    const issuerName = await addIssuer('RUN', feeIssuer);
+    await internalUnsafeImportPurse(issuerName, 'Zoe fees', false, feePurse);
```
Comment (katelynsills) [inline packages/dapp-svelte-wallet/api/src/lib-wallet.js:1655]: Nicely done!

File: packages/vats/src/bootstrap.js [hunk 3]
```diff
@@ -658,11 +658,17 @@ export function buildRootObject(vatPowers, vatParameters) {
           }),
         );
 
+        const userFeePurse = await E(zoe).makeFeePurse();
+        const zoeWUserFeePurse = await E(zoe).bindDefaultFeePurse(userFeePurse);
```
Comment (katelynsills) [inline packages/vats/src/bootstrap.js:662]: Nice!

File: packages/dapp-svelte-wallet/api/src/lib-wallet.js (hunk 1 above)
Comment (michaelfig) [reply packages/dapp-svelte-wallet/api/src/lib-wallet.js]: Yes.  There's an interleaving point before it.

Comment (michaelfig) [reply packages/dapp-svelte-wallet/api/src/lib-wallet.js]: It's better done as a function, so I've extracted `initIssuerToBoardId`.

**Review by katelynsills — COMMENTED** [review]
LGTM!

**Review by michaelfig — APPROVED** [review]

**Pull request merged**